data/assets/scene/solarsystem/heliosphere/bastille_day/fluxnodescutplane.asset

```diff
@@ -0,0 +1,88 @@
+asset.require("spice/base")
+local assetHelper = asset.require("util/asset_helper")
+local transforms = asset.require('scene/solarsystem/sun/transforms_heliosphere')
+--15 cmr emin01, 16 cmr emin03
+local TexturesPathEquitorial = asset.syncedResource({
+Type = "HttpSynchronization",
+    Name = "cutplanes_textures",
+    Identifier = "cutplanes_textures",
+    Version = 1
+})
+local TexturesPathMeridial = asset.syncedResource({
+    Type = "HttpSynchronization",
+    Name = "cutplane_meridial_textures",
+    Identifier = "cutplane_meridial_textures",
+    Version = 1
+})
+
+local Cutplane = {
+    Identifier = "Cutplane",
+    Parent = transforms.HNMReferenceFrame.Identifier,
+    --Parent = transforms.SunIAU.Identifier,
+    Transform = {
+    	--Rotation = {
```

Comment:
Lots of commented out things in this node. can they be removed? 

Anyhow, update according to previous comments on FixedRotations